pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -265,8 +299,138 @@ func (c *cyclonedxParser) GetPredicates(ctx context.Context) *assembler.IngestPr
 	return preds
 }
 
-func (s *cyclonedxParser) getPackageElement(elementID string) []*model.PkgInputSpec {
-	if packNode, ok := s.packagePackages[string(elementID)]; ok {
+func (c *cyclonedxParser) getVulnerabilities(ctx context.Context, cdxBom *cdx.BOM) error {
```

Comment:
dont need `cdxBom *cdx.BOM` here as its accessible from the method: https://github.com/guacsec/guac/blob/4ff4f13581c7f9850f39fcd2b0b9be93c208d7ed/pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go#L84-L91 (same for `getTopLevelPackage` and `getPackages`